sample/App.js

```diff
@@ -48,6 +48,34 @@ const App: () => React$Node = () => {
           )}
           <View style={styles.body}>
             <View style={styles.sectionContainer}>
+              <Text
+                style={styles.sectionTitle}
+                onPress={() => {
+                  Sentry.captureMessage('React Native Test Message');
+                }}>
+                captureMessage
+              </Text>
+              <Text
+                style={styles.sectionTitle}
+                onPress={() => {
+                  Sentry.captureException(new Error('captureException test'));
+                }}>
+                captureException
+              </Text>
+              <Text
+                style={styles.sectionTitle}
+                onPress={() => {
+                  throw new Error('throw new error test');
+                }}>
+                throw new Error
+              </Text>
+              <Text
+                style={styles.sectionTitle}
+                onPress={() => {
+                  Sentry.nativeCrash();
```

Comment:
L: don't need to be in this PR, but if `nativeCrash` is enabled, it means we are able to capture Java/Kotlin and C/C++ errors, would it make sense to have a sample for this too?